frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -54,6 +58,40 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 return match?.properties?.email || 'example@gmail.com'
             },
         ],
+        showSessions: [
```

Comment:
Super minor nit: since `showSessions` returns a simple feature flag check that will be removed when this is deployed to all users, it might not be worth it to create a separate selector here.